demo/nvflare/vertical/custom/trainer.py

```diff
@@ -77,13 +77,15 @@ def _do_training(self, fl_ctx: FLContext):
                 'gamma': 1.0,
                 'max_depth': 8,
                 'min_child_weight': 100,
-                'tree_method': 'approx',
+                'tree_method': 'hist',
                 'grow_policy': 'depthwise',
                 'objective': 'binary:logistic',
                 'eval_metric': 'auc',
             }
             if self._use_gpus:
-                self.log_info(fl_ctx, 'GPUs are not currently supported by vertical federated XGBoost')
+                if self._use_gpus:
```

Comment:
Is this condition duplicate? @rongou

Reply:
Ah good catch. Copy paste error. Sent #9578